src/Makefile

```diff
@@ -32,10 +34,20 @@ SCI_GEN_TARGET = ../lib/howl/scintilla.lua
 CFLAGS = -Wall -O2 -g $(LUAJIT_CFLAGS) $(SCINTILLA_CFLAGS) $(GTK_CFLAGS) -DHOWL_PREFIX=$(PREFIX)
 ARCHIVES = $(LUAJIT_ARCHIVE) $(SCINTILLA_ARCHIVE)
 LIBS = -lm -ldl ${GTK_LIBS} -lstdc++
-LD_FLAGS = -Wl,-E
+ifeq ($(UNAME_S),Darwin)
+	LD_FLAGS = -Wl,-export_dynamic -pagezero_size 10000 -image_base 100000000
+else
+	LD_FLAGS = -Wl,-E
+endif
 OBJECTS = main.o process_helpers.o
 DEP_OBJECTS = $(LPEG_OBJECT)
 
+ifeq ($(UNAME_S),Darwin)
```

Comment:
Since `md5` isn't used in the Makefile itself, I think it would be better to have this conditional directly in the `download` script.